{"name": "test_verify_phone_no[device1-phone_number1]", "status": "passed", "attachments": [{"name": "log", "source": "53a33431-549d-4f10-ab7d-fcfc015ea3eb-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "appium_driver", "value": "'device1'"}, {"name": "phone_number", "value": "['1234']"}], "start": 1638782736569, "stop": 1638782744752, "uuid": "ccf6d241-fe5c-4c17-b2ad-db4af5558e55", "historyId": "48534657ff2705363d6c47e6feb7c3e4", "testCaseId": "8f179043caacb0abcb24fe30a00081a0", "fullName": "test_cases.test_verify_ph_no.TestVerifyPhoneNo#test_verify_phone_no", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('log_on_failure', 'appium_driver')"}, {"name": "tag", "value": "flaky(reruns=3)"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_verify_ph_no"}, {"name": "subSuite", "value": "TestVerifyPhoneNo"}, {"name": "host", "value": "EPINHYDW042A"}, {"name": "thread", "value": "24716-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_verify_ph_no"}]}